src/main/java/com/webforj/samples/views/table/TableColumnAutoSizingView.java

```diff
@@ -84,50 +88,23 @@ private Table<MusicRecord> createTable() {
   }
 
   private void autoSizeAllColumns() {
-    table.setColumnsToAutoSize().thenAccept(ignored -> {
-    });
+    table.setColumnsToAutoSize().thenAccept(ignored -> {});
   }
 
   private void autoFitColumns() {
-    table.setColumnsToAutoFit().thenAccept(ignored -> {
-    });
+    table.setColumnsToAutoFit().thenAccept(ignored -> {});
   }
 
   private void autoSizeTitleColumn() {
-    Column<MusicRecord, ?> titleColumn = table.getColumns().stream()
-        .filter(col -> "Title".equals(col.getLabel()))
-        .findFirst()
-        .orElse(null);
-
-    if (titleColumn != null) {
-      table.setColumnToAutoSize(titleColumn).thenAccept(ignored -> {
-      });
-    }
+    table.setColumnToAutoSize(titleCol).thenAccept(ignored -> {});
   }
 
   private void resetToDefaults() {
```

Comment:
This repeats code from the createTable method. Refactor to remove the duplication please.